library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -67,14 +70,28 @@ internal constructor(
             return this
         }
 
+        /**
+         * Sets the sampling rate for Session Replay recorded Sessions. Please note that this
+         * sampling rate will be applied on top of already
+         *
+         * @param samplingRate the sampling rate must be a value between 0 and 100. A value of 0
+         * means no session will be recorded, 100 means all sessions will be recorded.
+         */
+        fun sessionReplaySampleRate(@FloatRange(from = 0.0, to = 100.0) samplingRate: Float):
```

Comment:
to be better aligned with recent renaming changes in the SDK v2, I would propose to add `set`:

```suggestion
        fun setSessionReplaySampleRate(@FloatRange(from = 0.0, to = 100.0) samplingRate: Float):
```

Also I'm not sure we need to add `SessionReplay` here, because it is clear already that we are building configuration for session replay.

Reply:
to align with Browser API 🤦 , it was specifically requested by product.

Reply:
Android SDK in v2 branch is using `set[XXX]SampleRate` naming. I'm fine with having `SessionReplay` in the name though.

Reply:
yes it makes sense to not have that in the signature. I will try to challenge this with the product.